packages/fxa-payments-server/server/config/index.js

```diff
@@ -111,6 +111,12 @@ const conf = convict({
       env: 'GEODB_ENABLED',
       format: Boolean,
     },
+    locationOverride: {
+      doc: 'override for forcing location',
+      format: Object,
+      default: {},
+      env: 'GEODB_LOCATION_OVERRIDE',
+    },
```

Comment:
Good idea to add this just as done in FXA-6883 for the auth server. Can we similarly update the payments server README for this config?

Reply:
done